web-console/Storage/OB-OB-Lifecycle-Management/OB-OBC-Lifecycle.md

```diff
@@ -0,0 +1,112 @@
+# OB/OBC Lifecycle Management Design
+Support object bucket (OB) and object bucket (OBC) CRUD
+
+## OBC
+
+### OBCs List
+The OBC list contains the following details: 
+* OBC name
+* Namespace
+* Status
+* Used
+* OBC Quota
```

Comment:
Can we drop "OBC" in "OBC Quota" since all of these other items are specific to the OBC too?

Reply:
This was requested by @julienlim, if it's more confusing that way we can definitely drop it.

Reply:
@lizsurette OBC Quotas are out of scope for 4.2 and the reason for asking for OBC quotas to be added in front is that these are not supported by native Kubernetes (nor the object provisioner library at the moment).  So, if you use the CLI, you'll not be able to use the native K8s CLI commands to see them at all, and if someone were to go look in K8s docs, they would not find it.

Having said this, if putting quotas in this design PR is going to prevent the merge, then I'd suggest we take out any references to quotas.

As noted above, OBC quotas are out of scope for 4.2 and so this is more about future direction/thoughts.